src/urbit/mar/chess/pals.hoon

```diff
@@ -0,0 +1,18 @@
+=,  format
+|_  pals=(set @p)
+++  grab
+  |%
+  ++  noun  pals
+  --
+++  grow
+  |%
+  ++  noun  pals
+  ++  json
+    %-  frond:enjs
+    :-  'mutuals'
```

Comment:
While `%pals` uses the term "mutuals", I don't think that we should:
1. It might be that we should be querying "targets" instead of "mutuals" (see above comment)
2. "Mutuals" doesn't mean anything to `%chess`, but it does to `%pals`. By returning `mutuals` as a mark for the scry, we're leaking information about what we use on the backend, and forwarding it to a user who may have no clue what the context is.
3. The mark will need to be changed, or else make no sense, if we ever change the backend (for example, to `%whom` instead of `%pals` - or even to the union of both)

We should go with something much more generic, like "friends" or "contacts"

Reply:
The idea was to keep things sort of modular and extensible, maybe in future someone wants to add more wires for pals/contacts to be handled in `%chess-pals`, but this makes sense. Let's go with `:-  'friends'`.

Reply:
Your explanation for your reasoning makes sense. My gut feeling is that it's better for us to define our own API and terminology, just so that we're not subject to changes in the app availability/design landscape (e.g. what apps will become popular and how they're written are out of our control, so let's just introduce our own idea of "friends" and we can adjust what that means on the backend as Urbit evolves).